### PR TITLE
Feature/facebook

### PR DESCRIPTION
**Pull request by Sschlenker**
As of now this would be a clean merge, but if my previous pull request is accepted this may cause and issue for the 575082f commit hash.  

Few things to note:
1.) I did not change the name of the html file to facebookscreenshot so if that will be an issue with how the website wants to reference this let me know.
2.) I still went through the usage of the module generator even though this is literally a single html page. 
3.) I left the main.scss in place, but since there is no actual css being used in the page I can rip that out entirely.  Let me know.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/CUModSquad/cu-ui/pull/5" target="_blank">ReviewNinja

File: facebook/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "cu-ui-facebook",
+  "version": "0.1.0",
+  "description": "Camelot Unchained UI - facebook",
+  "license": "MPL-2.0",
+  "main": "src/ts/main.tsx",
+  "contributors": [
+  ],
```
Comment (codecorsair) [inline facebook/package.json:11]: You may want to add yourself to the contributors on this module.

File: facebook/src/sass/main.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+body {
```
Comment (codecorsair) [inline facebook/src/sass/main.scss]: since this is empty, just delete the whole sass directory.

**Comment by codecorsair** [comment]
Looks good.  :+1:

File: facebook/package.json (hunk 1 above)
Comment (Sschlenker) [inline facebook/package.json:11]: !fixed

File: facebook/src/sass/main.scss (hunk 2 above)
Comment (Sschlenker) [inline facebook/src/sass/main.scss]: !fixed

**Comment by Sschlenker** [comment]
I made the changes you suggested.  For sake of ease I just added a new commit with these changes.  In the future would you prefer we do an interactive rebase to clean the history up or leave it and just add a new commit so we do not start messing with history?

**Comment by codecorsair** [comment]
I would like that we leave the history intact after you've already created PR.

**Pull request merged**